File: PHP/Mahasiswa.php
<?php
    include "Sivitas.php";
class Mahasiswa extends Sivitas
{
    private $nim = "";
    private $prodi = "";
    private $fakultas = "";
    function __construct($n, $na, $j, $u, $e, $ni, $p, $f)
    {
        $this->setNik($n);
        $this->setNama($na);
        $this->setJk($j);
        $this->setAsalUniv($u);
        $this->setEmailEdu($e);
        $this->nim = $ni;
        $this->prodi = $p;
        $this->fakultas = $f;
    }
    function setNim($a)
    {
        $this->nim = $a;
    }
    function setProdi($a)
    {
        $this->prodi = $a;
    }
    function setFakultas($a)
    {
        $this->fakultas = $a;
    }

    function getNim()
    {
        return $this->nim;
    }
    function getProdi()
    {
        return $this->prodi;
    }
    function getFakultas()
    {
        return $this->fakultas;
    }
    function __destruct(){
        
    }
}
?>

File: CPP/Mahasiswa.cpp
#include <iostream>
#include <string>
#include "Sivitas.cpp"

using namespace std;

class Mahasiswa : public Sivitas
{
private:
    string nim;
    string prodi;
    string fakultas;

public:
    Mahasiswa()
    {
        setNik("");
        setNama("");
        setJk("");
        setAsalUniv("");
        setEmailEdu("");
        nim = "";
        prodi = "";
        fakultas = "";
    };
    Mahasiswa(string a, string b, string c, string d, string e, string f, string g, string h)
    {
        setNik(a);
        setNama(b);
        setJk(c);
        setAsalUniv(d);
        setEmailEdu(e);
        nim = f;
        prodi = g;
        fakultas = h;
    }

    void setNim(string a)
    {
        nim = a;
    }
    void setProdi(string a)
    {
        prodi = a;
    }
    void setFakultas(string a)
    {
        fakultas = a;
    }

    string getNim()
    {
        return nim;
    }
    string getProdi()
    {
        return prodi;
    }
    string getFakultas()
    {
        return fakultas;
    }

    ~Mahasiswa()
    {
    }
};
